react/__mocks__/react-apollo.tsx

```diff
@@ -18,8 +18,23 @@ export function useQuery<TData, TVariables = OperationVariables>(
   options?: QueryHookOptions<TData, TVariables>
 ): Partial<QueryResult<TData, TVariables>> {
   return {
-    data: jsonSchema,
+    data: { documentPublicSchema: { schema: jsonSchema } },
     query: query,
     options: options,
   }
 }
+
+export function useMutation<TData = any, TVariables = OperationVariables>(
+  _: DocumentNode,
+  __?: QueryHookOptions<TData, TVariables>
+): Partial<MutationTuple<TData, TVariables>> {
```

Comment:
I think you don't need these parameters

```suggestion
export function useMutation<TData = any, TVariables = OperationVariables>(): Partial<MutationTuple<TData, TVariables>> {
```